content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -130,9 +127,8 @@ jobs:
             args:
             - -exc
             - |
-              mkdir -p ../ocw-hugo-themes/theme
-              tar -xvzf ../ocw-hugo-themes/ocw-hugo-themes-*.tgz -C ../ocw-hugo-themes/theme
-              hugo --config ../ocw-hugo-projects/((config-slug))/config.yaml --baseUrl /((base-url)) --themesDir ../ocw-hugo-themes/theme
+              cp ../webpack-json/webpack.json data
+              hugo --config ../ocw-hugo-projects/((config-slug))/config.yaml --baseUrl /((base-url)) --themesDir ../ocw-hugo-themes/
```

Comment:
What does `cp ../webpack-json/webpack.json data` do?  Isn't `webpack.json` already in the `ocw-hugo-themes` directory?

Reply:
No, it won't be.  `webpack.json` is generated at build time by an `AssetsPlugin` defined in the Webpack configuration and it's deposited in the theme's `base-theme/data` folder.  The `ocw-hugo-themes` folder in this case will also now just be the raw Github repo, cloned directly from github without any built artifacts.  The `webpack.json` file is uploaded to `ol-eng-artifacts` by the theme assets pipeline upon a completed build, and it has been added to the site pipeline definition as a versioned file.  The idea is that when a new version is detected, it will trigger site builds.  Although, once we merge your mass publish pipeline we'll want to remove `trigger: true` from it.  This line simply copies it from the resource import into the site's `data` file at build time so Hugo knows what URLs to write in.